ocs_ci/ocs/ui/page_objects/resource_list.py

```diff
@@ -0,0 +1,91 @@
+from selenium.webdriver.common.by import By
+
+from ocs_ci.ocs.exceptions import IncorrectUiOptionRequested
+from ocs_ci.ocs.ui.helpers_ui import format_locator
+from ocs_ci.ocs.ui.page_objects.searchbar import SearchBar
+from ocs_ci.ocs.ui.base_ui import logger
+
+
+class ResourceList(SearchBar):
+    """
+    Resource List module presented in OBC, OB, PVC, PV, BucketClass, BackingStore, StorageClass, VolumeSnapshotClasses,
+    BlockPools, etc.
+
+    This module is for selecting resource, navigation into, filtering, deletion, edition, etc.
+    """
+
+    def nav_to_resource(self, resource_name: str = None, resource_label: str = None):
+        """
+        Navigate to resource.
+        Important! Mandatory to pass either resource_name or resource_label
```

Comment:
Wouldn't it be a bit clearer to have 2 separate functions for navigation to resource by name and by label?

Reply:
done